src/test/java/io/mosip/mimoto/controller/InjiControllerTest.java

```diff
@@ -290,6 +301,36 @@ public void getIssuerConfigTest() throws Exception {
                 .andExpect(jsonPath("$.errors[0].errorMessage", Matchers.is(API_NOT_ACCESSIBLE_EXCEPTION.getMessage())));
     }
 
+    @Test
+    public void getIssuerWellknownTest() throws Exception {
+        String issuerId = "test-issuer";
+        String wellKnownResponse = "{\"issuer\":\"test-issuer\",\"authorization_endpoint\":\"https://example.com/auth\"}";
+        ResponseEntity<String> wellKnownResponseEntity;
+        wellKnownResponseEntity = restTemplate.getForEntity(wellKnownResponse, String.class);
```

Comment:
Line no 307 and 308 needed.
Steps to write this test,
1. Return mocked response for IssuerService wellKnown method.
2. Call IssuerController getIssuerWellknown method
3. Verify IssuerService method is invoked once.
4. assert the wellknown response